build_tools/buildkite/github/post_benchmarks_as_pr_comment.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+# Lint as: python3
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Posts benchmark results to GitHub as pull request comments.
+
+This script is meant to be used by Buildkite for automation. It requires the
+following environment to be set:
+
+- BUILDKITE_BUILD_URL: the link to the current buildkite build.
```

Comment:
Nit: channeling Scott

```suggestion
- BUILDKITE_BUILD_URL: the link to the current Buildkite build.
```